ovos_core/intent_services/dispatcher.py

```
@@ -0,0 +1,255 @@
+        self._in_flight: Dict[str, List[_InFlightDispatch]] = {}
+        self._lock = threading.Lock()
+        # framework done-signals (legacy namespace; do NOT bridge to the spec trio)
+        self.bus.on("mycroft.skill.handler.complete", self._on_skill_complete)
+        self.bus.on("mycroft.skill.handler.error", self._on_skill_error)
+
+    def shutdown(self):
+        try:
+            self.bus.remove("mycroft.skill.handler.complete", self._on_skill_complete)
+            self.bus.remove("mycroft.skill.handler.error", self._on_skill_error)
+        except Exception:
+            pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)